package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "bluebird": "~2.9.25",
-    "fuel-auth": "~1.0.0",
+    "fuel-auth": "2.1.0",
```

Comment:
Yeah, I let that fall by the wayside. if you guys want to do it quick and dirty with the tests passing, I have a few days this week and could implement it in the way I was hoping to (with better promise/callback integration and such).